portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -78,8 +78,13 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-9183 | Test Summary: Current AC plan is collapsible."
+	@ignore = "true"
 	@priority = 3
 	test CollapseCurrentPlan {
+
+		// AC Bug: Current plans card is missing
+		// AC Bug ticket: LRAC-14001
+
```

Comment:
Hey @joshitagaki, I see this test is passing constantly in the release routine, why should we disable it? I think we can continue with this case.

[History](https://testray.liferay.com/home/-/testray/case_results/2345384491/history?tab=result&hideFilterPin=true)

Reply:
The bug was caused by another fix by Adriano and is not yet present in the release branch

Reply:
It's still failing consistently on upstream until the bug is fixed. Should we leave it and wait until it's fixed or do you think it makes sense to ignore it on upstream until it is fixed?

Reply:
Adriano has already sent the [PR](https://github.com/liferay/com-liferay-osb-faro-private/pull/4708) of this fix for 7.1.x and Marcellus has already included this fix in the UAT, so it will continue to work in the release routine and it will work again in the upstream routine when the PR is merged.

Let's follow the correct flow and disable the test as you suggested and when the bug is validated we can enable it again

Could you put a comment reminding us to enable the test again in the bug ticket?